src/main/kotlin/racinggame/domain/Game.kt

```diff
@@ -0,0 +1,22 @@
+package racinggame.domain
+
+class Game(carNumber: Int, initMoving: Int) {
+    val totalGameTurn = initMoving
+    val cars: List<RacingCar> by lazy {
+        makingCars(carNumber)
+    }
+
```

Comment:
```suggestion
class Game(val cars: List<RacingCar>, val totalGameTurn: Int) {
..
```

Game을 만들 때 파라미터를 적극 활용한 부분은 좋아요! 하지만, 파라미터로 생성이 되는 클래스의 단점은 테스트를 하기 어렵다고 봐요.

아마, Game에 임의로 움직인 자동차를 넣으려고 할 때, 리플랙션을 이용하여 cars를 수정하게 될거에요. 과연 리플랙션을 이용한 테스트가 올바른 테스트인가도 저는 고민이 많이 되더라구요. 

이러한 테스트를 피하기 위해서 제가 제시한 코드처럼 프로퍼티로 받도록 만들고, 파라미터를 통해서 Game을 만드는 부분은 팩토리 메서드 패턴을 사용하여 하는게 테스트 + 생성에 용이하다고 봐요. 

- 팩토리 메서드 패턴? https://kyucumber.github.io/posts/book/effective-kotlin/effective-kotlin-item-33